src/Annotator.tsx

```diff
@@ -938,10 +939,16 @@ export class Annotator extends React.Component<Props, State>{
         return data;
     };
 
+    onReload = ()=> {
+        location.reload();
+    }
+
     onUpload = () => {
         if (this.props.asyncUpload == null) {
             return;
         }
+        if(confirm("确定要提交吗")==false)
```

Comment:
I don't think this is a handy feature. It may slow things down so much. BTW, if you create a feature like this, please add a configurable param in the `props` to turn it off.

A better way to do this is to cache the uploaded images info and allow users to cancel. You can do it in the back-end and the parent component. 

You can also implement this cancelable annotation in this project, but it requires more work. 
 
1. You need to create a queue to cache annotated images. And you only invoke `props.upload` when the queue is full or after a certain timeout, like containing more than 10 annotations.
2. Because you still need to switch images, and we do not call the `props.asyncUpload` immediately. So you need to separate the `props.asyncUpload` into `props.upload` and `props.onAnnotated`.